helios/models.py

```diff
@@ -244,6 +244,22 @@ def get_by_short_name(cls, short_name):
       return cls.objects.get(short_name=short_name)
     except cls.DoesNotExist:
       return None
+    
+  def save_questions_safely(self, questions):
+    """
+    Because Django doesn't let us override properties in a Pythonic way... doing the brute-force thing.
+    """
+    # verify all the answer_urls
+    for q in questions:
+      for answer_url in q['answer_urls']:
+        if not answer_url or answer_url == "":
+          continue
+          
+        # abort saving if bad URL
+        if not (answer_url.startsWith("http://") or answer_url.startsWith("https://")):
+          return
```

Comment:
wouldn't it be nicer if you just remove the bad url instead of the whole question? For instance, I can add a question with  no option... so why couldn't I get the question with just the bad url removed?